Makefile

```diff
@@ -1109,8 +1109,8 @@ $(foreach ut, $(ALL_UNITTESTS), $(eval $(call UT_TEMPLATE,$(ut))))
 
 .PHONY: python_ut_test
 python_ut_test:
-	$(V0) @echo "  PYTHON_UT test.py"
-	$(V1) python/test.py
+	$(V0) @echo "  PYTHON_UT integrationtests-basic"
+	$(V1) python/integrationtests-basic
```

Comment:
Can we add `-v`? That would make it a lot easier to see what output goes with which tests.

Reply:
yes we can :D